package.json

```diff
@@ -29,6 +29,7 @@
   },
   "devDependencies": {
     "babel-plugin-external-helpers": "6.22.0",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.13",
```

Comment:
I can lock down these version numbers if you would like. Also, we could add an `.npmrc` with `save-exact=true` so that this preference would apply to all future contributors. What do you think?

Reply:
I do prefer exact versions. I think I set that on my CLI config a while back but didn't know about the `.npmrc` option. That is good to know. Locking these versions down and adding the `.npmrc` would be a great addition here!